src/__test_resources__/legacyUIFolder.ts

```diff
@@ -488,6 +488,32 @@ export const legacyUIFolder = {
                 canWrite: true,
               },
             },
+            "0xb": {
```

Comment:
Fun fact, if the id is not wrapped with quotes, then at runtime it is interpreted as an hexadecimal number, and the id becomes `11` 💩